src/Service/CloudWatchLogs/src/Input/PutLogEventsRequest.php

```diff
@@ -162,9 +162,13 @@ private function requestBody(): array
             throw new InvalidArgument(sprintf('Missing parameter "logStreamName" for "%s". The value cannot be null.', __CLASS__));
         }
         $payload['logStreamName'] = $v;
+        if (null === $v = $this->logEvents) {
+            throw new InvalidArgument(sprintf('Missing parameter "logEvents" for "%s". The value cannot be null.', __CLASS__));
```

Comment:
Is it a BC break?.. The field is required, but previously AsyncAws replaced null by empty array.